content/en/blog/2025/ambient-multicluster/index.md

```diff
@@ -0,0 +1,106 @@
+
+Multicluster has been one of the most requested ambient features — and as of Istio 1.27, it's now available.
+We sought to capture the benefits and avoid the complications of multicluster architectures using the same modular design that ambient users love.
+While still in alpha, this release delivers the core functionality of a multicluster mesh and lays the groundwork for a full feature set in upcoming releases.
+
+## Multicluster's Many Benefits (and Challenges)
+
+Multicluster architectures increase outage resilience, shrink the blast radii,
+ease adoption of data residence policies, and simplify cost tracking.
+That said, integrating multiple clusters poses connectivity, security, and operation hurdles.
+
+In a single Kubernetes cluster, every pod can directly connect to another pod via a unique pod IP or service VIP.
+We lose these guarantees when we start thinking of multicluster architectures.
+IP address spaces of different clusters might overlap.
+Even if they didn't, nodes in one cluster would not know how to route traffic from one cluster to another.
+
+Establishing cross-cluster connectivity also presents security challenges.
+Cross-cluster connectivity means that pod-to-pod traffic can leave cluster boundaries -- and that pods may accept connections from outside the cluster.
+Without care, an attacker could connect to a vulnerable pod, or sniff unencrypted traffic.
+
+For a multicluster solution to be viable, it must at least securely connect clusters, and do so
+through APIs that are simple enough to keep pace with ever-changing environments.
+
+## Key Components.
+
+Ambient multicluster extends ambient with new components and minimal APIs to
+securely connect clusters using the same lightweight, modular architecture.
+
+### East-West Gateways
+
+Each cluster deploys an east-west gateway with a globally routable IP that acts as an entrypoint for cross-cluster communication.
+The east-west gateways are configured using GatewayAPI and controlled by istiod.
+A ztunnel communicates across clusters by connecting to the remote cluster's east-west gateway and sending the destination service FQDN.
+The east-west gateway will then forward the connection to a cluster-local pod of its choosing.
+As such, overlapping IP spaces are of no concern because we never directly address a pod in a remote cluster.
+Ambient multicluster achieves cross-cluster connectivity without changes to cluster networking configuration.
+We can achieve this connectivity using only ambient and declarative APIs.
+There is no need to restart workloads, manage IP address spaces, or configure routing tables.
+
+### Double HBONE
+
+Ambient Multicluster uses nested [HBONE](https://istio.io/latest/docs/ambient/architecture/hbone/) connections to secure traffic traversing cluster boundaries while preserving ambient's strong security.
+An outer HBONE connects the source ztunnel to its east-west gateway while an inner HBONE tunnel extends the connection to the destination.
```

Comment:
Is this sentence necessary when the next two break down the inner and outer tunnels again?